src/Corax/IndexSearcher/MultiTermMatches/IndexSearcher.MultiTermMatch.Simple.cs

```diff
@@ -119,4 +121,18 @@ public MultiTermMatch ExistsQuery<TScoreFunction>(string field, TScoreFunction s
     {
         return MultiTermMatchBuilder<TScoreFunction, ExistsTermProvider>(field, null, scoreFunction, false, Constants.IndexSearcher.NonAnalyzer);
     }
+
+    public MultiTermMatch RegexQuery<TScoreFunction>(string field, TScoreFunction scoreFunction, Regex regex)
+        where TScoreFunction : IQueryScoreFunction
+    {
+        var fields = _transaction.ReadTree(Constants.IndexWriter.FieldsSlice);
+        using var _ = Slice.From(Allocator, field, ByteStringType.Immutable, out var fieldName);
```

Comment:
You don't use `fieldName` anywhere. Do you to do this slice?

Reply:
Oh, didn't catch it, thx.